### ISSUE TITLE
[Panel] section: files doesn't take the template value

### ISSUE THREAD

**Issue by beebmx** [getkirby/kirby#793]
Using the `beta-3` the section `type:files` doesn't take the options from the template.

Here is the blueprints:

```yml
title: Home

sections:
  files:
    headline: Files
    type: files
    template: cover
    layout: list
```
Template:
```yml
title: Cover

fields:
  alt:
    label: Alt Text
    type: text
```

The result is: 
<img width="686" alt="2018-08-01 19 06 43" src="https://user-images.githubusercontent.com/2191576/43555163-37ba706e-95be-11e8-8aeb-04f7c3bd6925.png">

If the template line is removed, the result is:
<img width="668" alt="2018-08-01 19 09 15" src="https://user-images.githubusercontent.com/2191576/43555213-7af7ba12-95be-11e8-98d4-df5661df9fc8.png">

**Comment by distantnative** [comment]
I think that happens because those files do not have the template `cover`.

```
sections:
  files:
    headline: Files
    type: files
    template: cover
    layout: list
```

That section setting only lists files with the `cover` template.

**Comment by gerricom** [comment]
I ran into the same issue and it's a bit irritating when you are in a Kirby 2-like workflow:

I set up a page blueprint and started to insert some content and some files. Later I thought about getting some extra fields for files and I built a file blueprint and added the `template` in my page blueprint which caused all files to vanish. I now understand, that this is correct because I changed the template in the blueprint which filters files. But there is no option yet to change a template of a file - or am I missing something?
